R/load-data.R

```diff
@@ -11,15 +11,14 @@ source(here::here("R", "load-public-data.R"))
 source(here::here("R", "load-private-data.R"))
 source(here::here("R", "munge-data.R"))
 
-load_data <- function(open_data = TRUE, data_type = "specimen",
-                      start_date = as.Date("2021-11-20")) {
+load_data <- function(open_data = TRUE, ...) {
```

Comment:
May error when running the original analysis as `date_type` is used inside the if loop (though doesn't need to be so can just drop that line)